ProductsAndCustomerCrudAsync/Controllers/CustomersController.cs

```diff
@@ -0,0 +1,99 @@
+﻿using EFPractice.Data;
+using EFPractice.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace EFPractice.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+   
+    public class CustomersController : ControllerBase
+    {
+        private readonly PrjContext _context;
+
+        public CustomersController(PrjContext context)
+        {
+            _context = context;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Customer>>> GetCustomers()
```

Comment:
I request you to please make a note here ActionResult is base class, IActionResult is it's Parent Interface, so better to go with IActionResult